backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/AvailabilityService.cs

```diff
@@ -328,15 +334,17 @@ public async Task<bool> ValidateLinkAsync(long? slotId, string slotLink)
                 .FirstOrDefaultAsync(s => s.Link == link);
         }
 
-        private async Task SaveEmailTemplateConfig(EmailTemplatesSettingsDto settingsDto, AvailabilitySlot slot)
+        private async Task SaveEmailTemplateConfig(ICollection<EmailTemplatesSettingsDto> settingsDto, AvailabilitySlot slot)
         {
-            var emailTemplate = _mapper.Map<EmailTemplate>(settingsDto,
-                    opts => opts.AfterMap((_, dest) =>
-                    {
-                        dest.AvailabilitySlot = slot;
-                    }));
-            await _context.EmailTemplates.AddAsync(emailTemplate);
-            slot.EmailTemplates.Add(emailTemplate);
+            var emailTemplates = settingsDto.Select(s => _mapper.Map<EmailTemplate>(s,
+                opts => opts.AfterMap((_, dest) =>
+                {
+                    dest.AvailabilitySlot = slot;
+                }))).ToList();
+            
+            await _context.EmailTemplates.AddRangeAsync(emailTemplates);
```

Comment:
after AddRangeAsync we dont need await _context.SaveChangesAsync(); ?  hm

Reply:
_context.SaveChangesAsync() calls in methods where we create/update Availability Slot